Symfony/CS/Tests/Fixer/PSR2/FunctionCallSpaceFixerTest.php

```diff
@@ -80,6 +80,15 @@ public function testFixProvider()
             array(
                 '<?php $a = 2 * (1 + 1);',
             ),
+            array(
+                '<?php echo ($a == $b) ? "foo" : "bar";',
+            ),
+            array(
+                '<?php echo ($a == test($b)) ? "foo" : "bar";',
+            ),
+            array(
+                '<?php include ($html)? "custom.html": "custom.php";',
```

Comment:
It is not correct. If i check only `T_STRING` following code failure. 
```
 array(
    '<?php echo isset($name) ? "y" : "n";',
    '<?php echo isset ($name) ? "y" : "n";',
),
```
`isset` is not `T_STRING` but it is function.
So we need list of tokens to detect real function or not.